payload/reggae/rules/cmake.d

```diff
@@ -0,0 +1,320 @@
+module reggae.rules.cmake;
+
+imported!"reggae.build".Target[] cmakeBuild(imported!"reggae.types".ProjectPath projectPath,
+                                            imported!"reggae.rules.dub".Configuration cmakeConfig = imported!"reggae.types".Configuration("Release"),
```

Comment:
This should be probably moved to `reggae.types`, but I think it would be better to do it in a follow-up PR.

Reply:
(Did it in a second commit.)